master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -43,10 +43,21 @@ func (a *agentResourceManager) Receive(ctx *actor.Context) error {
 		}
 
 	case AllocateRequest:
+		// armand: this code exists to handle the case where an experiment does not have
```

Comment:
Is there any todo in the future? If not, you don't have to specifically leave your name on it because the comments are already self-container.